seed-job/buildConfiguration.yaml

```diff
@@ -0,0 +1,17 @@
+builds:
+  - name: Rest-Gateway
+    path: client/rest
+  - name: Catbuffer-Parser
+    path: catbuffer/parser
+  - name: Sdk-Python
+    path: sdk/python
+    dependsOn:
+      - catbuffer/parser
+  - name: Sdk-Javascript
+    path: sdk/javascript
+    dependsOn:
+      - catbuffer/parser
+  - name: Jenkins
+    path: jenkins
+  - name: Linters
+    path: linters
```

Comment:
should everything depend on linters and/or jenkins? 🤔

Reply:
maybe after we get to a stable state?